Makefile

```diff
@@ -0,0 +1,12 @@
+run-lint:
+	cd src/apps/backend && pipenv run mypy --config-file mypy.ini .
```

Comment:
Can we use 2 spaces here and other places?

Reply:
Makefile only allowing us to indent using tab, not sure there is any doable way to define make rule via space indentation